resolwe/flow/models/storage.py

```diff
@@ -5,6 +5,31 @@
 from django.db import models
 
 from .base import BaseModel
+from .functions import JsonGetPath
+
+
+class StorageManager(models.Manager):
+    """Manager for Storage objects."""
+
+    def with_json_path(self, path, field='result'):
```

Comment:
It would be more useful if `field` is generated from `path`, so you can call method multiple times (without manually defining `field`).

Reply:
I thought about it, but the question is how to make this the most intuitive. Perhaps we could name the resulting field something like `'_'.join(['json'] + path)`? The only problem with this is that currently the path parsing logic is inside the projection function, while this would require path processing here.